static/docs/commands-reference/commit.md

```diff
@@ -49,23 +49,23 @@ DVC can't guarantee reproducibility in those cases – You commit any data you
 want. Let's take a look at what is happening in the fist scenario closely:
 
 Normally DVC commands like `dvc add`, `dvc repro` or `dvc run` commit the data
-to the <abbr>DVC cache</abbr> after creating a DVC-file. What _commit_ means is
-that DVC:
+to the <abbr>cache</abbr> after creating a DVC-file. What _commit_ means is that
+DVC:
 
 - Computes a checksum for the file/directory
 - Enters the checksum and file name into the DVC-file
 - Tells the SCM to ignore the file/directory (e.g. add entry to `.gitignore`)
   (Note that if the <abbr>workspace</abbr> was initialized with no SCM support
   (`dvc init --no-scm`), this does not happen.)
-- Adds the file/directory or to the DVC cache
+- Adds the file/directory or to the cache directory
```

Comment:
Why should we explicitly specify that this is a directory?

Reply:
it sounds not like we always just copy files to some directory, while it's more complicated

Reply:
I would try to use cache directory when we explaining it's structure, or some internal and it's clear from the context that there is some directory indeed

Reply:
Hmmm good points. I was just reviewing the DVC "branding" of terms like DVC project and DVC remote, trying to avoid it sometimes. I'm going to just reduce this term to "cache" or some times "project's cache" except when explaining cache internals or when related to the `dvc cache dir` command (and corresponding config option).

Reply:
Done! (Along with related improvements) in 3f7884b.